Unity-Technologies/Graphics
# Make AOV API available at runtime

### Purpose of this PR
Make AOV API available at runtime. Graphic compositor and Automative configurator need it.

Fix fogbugz: https://fogbugz.unity3d.com/f/cases/1265070/
Fix https://fogbugz.unity3d.com/f/cases/1265001/

This PR:
- Add marker for all AOV request operation on GPU
- Add remapping options for Depth Pyramid debug view mode
![depth mode](https://user-images.githubusercontent.com/21238527/89471658-056bf100-d77f-11ea-8095-89e398d97b5f.gif)
- Fixed AOV export of depth buffer which now correctly export linear depth (case 1265001)
- Add an option to support AOV shader at runtime in HDRP settings (case 1265070) (by not stripping the debug display shader)
- Added error message when allocation from AOV callback allocator fail.

---
### Testing status

QA: 
tested manually that I can build a player for graphic compositor, without the check box it fail to render correctly.
Tested imodifying HDRP_Test 3003_Lightmode to export linear depth. All work correctly.
Automated test for depth buffer export to come in a future PR.

No futher QA required.

Yamato: https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics/tree/HDRP%252Fenable-aov-api-at-runtime/.yamato%252Fall-hdrp.yml%2523All_HDRP_fast-trunk/3029313/job/pipeline

---
### Comments to reviewers
Notes for the reviewers you have assigned.
